model/vcs/metrics.yaml

```diff
@@ -0,0 +1,139 @@
+groups:
+  - id: metric.vcs.repository.change.count
+    type: metric
+    metric_name: vcs.repository.change.count
+    brief: 'The number of changes (pull requests) in a repository, categorized by their state (either open or merged)'
+    instrument: updowncounter
+    unit: "{change}"
+    stability: experimental
+    attributes:
+      - ref: vcs.change.state
+        requirement_level: required
+      - ref: vcs.repository.url.full
+        requirement_level: required
+  - id: metric.vcs.repository.change.time
+    type: metric
+    metric_name: vcs.repository.change.time
+    brief: 'The time distribution a change (pull request) has been in open state or that it took to go from open to another state.'
+    instrument: histogram
+    unit: "s"
+    stability: experimental
+    attributes:
+      - ref: vcs.repository.url.full
+        requirement_level: required
+      - ref: vcs.repository.ref.name
+        requirement_level: required
+      - ref: vcs.change.state
+        requirement_level: required
+  - id: metric.vcs.repository.change.time_to_approval
+    type: metric
+    metric_name: vcs.repository.change.time_to_approval
+    brief: 'The amount of time it took a change (pull request) to go from open to approved'
+    instrument: histogram
+    unit: "s"
+    stability: experimental
+    attributes:
+      - ref: vcs.repository.url.full
+        requirement_level: required
+      - ref: vcs.repository.ref.name
+        requirement_level: required
+  - id: metric.vcs.repository.count
+    type: metric
+    metric_name: vcs.repository.count
+    brief: 'The number of repositories in an organization'
+    instrument: gauge
+    unit: "{repository}"
+    stability: experimental
+    attributes: []
+  - id: metric.vcs.repository.ref.count
+    type: metric
+    metric_name: vcs.repository.ref.count
```

Comment:
not: same comment on naming - does `repository` in the name add any value? `vsc.ref.count` seems to be as precise and expressive, but shorter and easier to find

Reply:
We discussed this in today's CICD SIG meeting.

Like the rename of `change` we did decide to make this rename of `vsc.repository.ref.*` to `vsc.ref.*`.

Done in cfffb4f8cd4f40023255624e6a4ce56dcfcb0d74.